package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@internetarchive/your-webcomponent",
-  "version": "0.0.0",
+  "name": "@internetarchive/throttler-debouncer",
```

Comment:
It won't let me comment on the line down below, but in the `dependencies` section of `package.json`, you no longer need `lit-element` and `lit-html` as `dependencies`. They can be moved into `devDependencies` since they're only being used to demo the throttler / debouncer. 

You might get a warning when you do that, but it's okay. I think there's a setting we can fix that with.

Reply:
updated